### ISSUE TITLE
pfacct data doesn't get hourly aggregated anymore

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5300]
**Describe the bug**
The bandwidth_accounting data doesn't get hourly aggregated anymore. 

It's currently 12:52PM and here are my rows:
```
| 445146671383899 | 1313146747820393572 | 2020-04-06 11:10:00 |       10 |        10 | 94:db:c9:38:85:5b |         1 |         1 | 2020-04-06 12:50:58 |          20 |
| 445146671383899 | 1313146747820393572 | 2020-04-06 11:15:00 |       10 |        10 | 94:db:c9:38:85:5b |         1 |         1 | 2020-04-06 12:51:02 |          20 |
| 445146671383899 | 1313146747820393572 | 2020-04-06 11:20:00 |       10 |        10 | 94:db:c9:38:85:5b |         1 |         1 | 2020-04-06 12:51:06 |          20 |
```

The expectation is that these get aggregated into a `2020-04-06 11:00:00` bucket **inside the bandwidth_accounting table**

Otherwise the bandwidth_accounting table will indeed contain data from the last 24 hours but it will be with buckets of 5 minutes.

This was previously working and regressed at some point, not sure when

**Comment by jrouzierinverse** [comment]
It was that way from the beginning.

**Comment by jrouzierinverse** [comment]
From the very first commit

**Comment by julsemaan** [comment]
It was working at some point in the feature/pfacct branch as it was part of my tests

Also I emphasized the importance of this functionality in multiple comments in the pfacct PR so I don't understand why it never worked

So please get what is described in this comment working:
https://github.com/inverse-inc/packetfence/pull/5181#issuecomment-599628182

**Comment by jrouzierinverse** [comment]
After a conversation with @julsemaan this issue was introduced by using last_updated for on/off/unknown online status.
Also by increasing the aggregation window from 2hr to 24hr.
The issues brought up here will be addressed in #5302 as the real issue is the expiration of RADIUS sessions.
